### DISCUSSION TITLE
something about FetchAPI

### DISCUSSION THREAD

**Discussion by 2382958980** [zepp-health/zeppos-forum#286] [Q&A]
## Apology
Honestly I don't familiar with FetchAPI or something about HTTP requests. So this question might be confusing, I'm sorry for that but I don't know how to continue. This problem doesn't seem to have much to do with zeppOS, I don't really know where the problem lies. if this question is Inappropriate to be asked at here, I will delete it as soon as possible. 

## Question Description
I have searched relevant documents and downloaded samples program of FetchAPI. And when I use it to POST to postbin, postbin did recieved my text. So I tried modifying the `url` to my own server (http://[cantiane.top/heartrate.php](http://cantiane.top/heartrate.php) This page can be opened normally), 
the php code of this is here:
![U{DAJ6{4(U @}NM2VLG_CG5](https://github.com/zepp-health/zeppos-forum/assets/59905973/95217d18-bc5b-4c3a-a47e-b6840a028283)
I tried to post using this (same as the sample provided):
![Z8A%RL) OG~(Z8ECU~P%7)5](https://github.com/zepp-health/zeppos-forum/assets/59905973/07bde423-0ea9-4f3c-b5c1-7d70e3c6625a)
but when I click the button on my device, the server has no react(it should print'POST SUCCESS' in my expectation and the network is connected normally）

Please let me know if you need any additional information.

**Comment by MyoungXUE** [comment]
Can you run Zepp OS Simulator normally? I suggest simulating it in the emulator. If the request fails, you can find the error message in the Console Network panel.

<img width="1009" alt="image" src="https://github.com/zepp-health/zeppos-forum/assets/107526244/b7f70fcf-b56d-44f8-87a1-8faac5ac68b6">

**Reply by 2382958980** [reply]
In fact, I am too eager for success. I should learn about HTTP first instead of asking questions here without understanding anything. Thank you all and apologize. Thanks again.

**Comment by htoooth** [comment]
@2382958980 

Please provide the code where the problem occurred, including the server-side. Could it be a problem with HTTPS and HTTP?

**Reply by 2382958980** [reply]
In fact, I am too eager for success. I should learn about HTTP first instead of asking questions here without understanding anything. Thank you and apologize. Thanks again.
